pkg/util/util.go

```diff
@@ -442,3 +516,37 @@ func ComputeHash256(yamlFile string) (string, error) {
 	sha := base64.URLEncoding.EncodeToString(hasher.Sum(nil))
 	return sha, nil
 }
+
+func ReloadCheCluster(client client.Client, cheCluster *orgv1.CheCluster) error {
```

Comment:
It is already here 
https://github.com/eclipse-che/che-operator/blob/main/pkg/deploy/checluster.go#L81

Reply:
I cannot use that version, because it requires deploy context which is main controller specific.